common/util.go

```diff
@@ -52,14 +52,15 @@ func ParseConnectorMetadata() (ConnectorMetadata, error) {
 }
 
 // HandleHTTPRequest sends message and headers data to HTTP endpoint using POST method and returns response on success or error in case of failure
```

Comment:
Can you add the description about what int represents? -1 meaning error and others, if any?